libr/include/r_core.h

```diff
@@ -209,6 +209,33 @@ typedef struct r_core_visual_t {
 	RCoreVisualMode printidx;
 	RList *tabs;
 	int tab;
+	bool textedit_mode;
+	int obs;
+	bool __ime;
+	int __nib;
+	bool __imes;
+	int blocksize;
+	bool autoblocksize;
+	int disMode;
+	int hexMode;
+	int printMode;
+	bool snowMode;
+	RList *snows;
+	int color;
+	int zoom;
+	int currentFormat;
+	int current0format;
+	char numbuf[32];
+	int numbuf_i;
+	bool splitView;
+	ut64 splitPtr;
+	int current3format;
+	int current4format;
+	int current5format;
+	RConfigHold *hold;  // XXX should be a tab-specific var
```

Comment:
wanna try to make this change, also this is more like TODO than XXX

Reply:
The comment is very vague for me right now, I'd need more explanation. I can make it a TODO :)